minimint/src/db.rs

```diff
@@ -3,12 +3,15 @@ use crate::transaction::Transaction;
 use minimint_api::db::DatabaseKeyPrefixConst;
 use minimint_api::encoding::{Decodable, Encodable};
 use minimint_api::{PeerId, TransactionId};
+use minimint_core::epoch::EpochHistory;
 use std::fmt::Debug;
 
 pub const DB_PREFIX_PROPOSED_TRANSACTION: u8 = 0x01;
 pub const DB_PREFIX_ACCEPTED_TRANSACTION: u8 = 0x02;
 pub const DB_PREFIX_DROP_PEER: u8 = 0x03;
 pub const DB_PREFIX_REJECTED_TRANSACTION: u8 = 0x04;
+pub const DB_PREFIX_EPOCH_HISTORY: u8 = 0x05;
+pub const DB_PREFIX_LAST_EPOCH: u8 = 0x06;
```

Comment:
Remember to update `database.md`

Reply:
Good point, would be nice to have an audit for that